src/instruments/src/ISIS/index.tsx

```diff
@@ -31,6 +31,9 @@ export const ISISDisplay: React.FC = () => {
 
     useInteractionEvent('A32NX_ISIS_KNOB_CLOCKWISE', () => {
         lastPilotInput.current = Date.now();
```

Comment:
For anyone else reviewing, the XML only sends the event if bugs mode is active... 😅